src/api/Feed.js

```diff
@@ -518,13 +519,17 @@ class Feed extends Base {
 
         const versionsPromise = shouldShowVersions ? this.fetchVersions() : Promise.resolve();
         const currentVersionPromise = shouldShowVersions ? this.fetchCurrentVersion() : Promise.resolve();
+
+        const filteredActivityTypes = [];
+        if (shouldShowAnnotations && permissions[PERMISSION_CAN_VIEW_ANNOTATIONS]) {
+            filteredActivityTypes.push(FILE_ACTIVITY_TYPE_ANNOTATION);
+        }
+        if (shouldShowAppActivity) filteredActivityTypes.push(FILE_ACTIVITY_TYPE_APP_ACTIVITY);
+        filteredActivityTypes.push(FILE_ACTIVITY_TYPE_COMMENT);
```

Comment:
would `PERMISSION_CAN_COMMENT` be checked for `FILE_ACTIVITY_TYPE_COMMENT`, similar to annotations?

Reply:
Good point, will do to keep consistent